schema_test.go

```diff
@@ -137,6 +137,48 @@ func TestSchemaOf(t *testing.T) {
 			required binary value (STRING);
 		}
 	}
+}`,
+		},
+
+		{
+			value: new(struct {
+				A map[int64]string `parquet:",optional" parquet-value:",json"`
+			}),
+			print: `message {
+	optional group A (MAP) {
+		repeated group key_value {
+			required int64 key (INT(64,true));
+			required binary value (JSON);
+		}
+	}
+}`,
+		},
+
+		{
+			value: new(struct {
+				A map[int64]string `parquet:",optional"`
+			}),
+			print: `message {
+	optional group A (MAP) {
+		repeated group key_value {
+			required int64 key (INT(64,true));
+			required binary value (STRING);
```

Comment:
I guess I'd expect the ",optional" bit to make this field not "required"? Or does optional imply something else?

Reply:
if you want the value field to be optional you will need :
```
			value: new(struct {
				A map[int64]string `parquet:",optional" parquet-value:",optional"`
			}),
```